pyccel/ast/literals.py

```diff
@@ -0,0 +1,134 @@
+""" This module contains all literal types
+"""
+from sympy               import Integer as sp_Integer
+from sympy               import Float as sp_Float
+from sympy.logic.boolalg import BooleanTrue as sp_BooleanTrue, BooleanFalse as sp_BooleanFalse
+from sympy.core.expr     import Expr
+
+from .basic              import PyccelAstNode, Basic
+from .datatypes          import (NativeInteger, NativeBool, NativeReal,
+                                  NativeComplex, NativeString, default_precision)
+
+__all__ = (
+    'LiteralBooleanTrue',
+    'LiteralBooleanFalse',
+    'LiteralInteger',
+    'LiteralFloat',
+    'LiteralComplex',
+    'LiteralImaginaryUnit',
+    'LiteralString',
+    'get_default_literal_value'
+)
+
+#------------------------------------------------------------------------------
+class LiteralBooleanTrue(sp_BooleanTrue, PyccelAstNode):
```

Comment:
What do you think of calling this class `LiteralTrue`, and the one below `LiteralFalse`? There is no ambiguity, I believe.

Reply:
Sounds good to me